docs/samples/logger/local/README.md

```diff
@@ -0,0 +1,109 @@
+# Local Test of Kfserving Logger
+
+In one terminal start an echo http server on port 8000
+
+```
+docker run -it -p 8000:80 --rm -t mendhak/http-https-echo
+```
+
+Start an SKLearn Iris model on port 8081
+
+```
+python -m sklearnserver --model_dir gs://kfserving-samples/models/sklearn/iris --model_name sklearn-iris --http_port 8081
```

Comment:
Can you make sure the path to sklearnserver is relative to this readme?

Reply:
Its assuming you have pip installed it so you don't need a path. Will add note to README.